libs/aws/odc/aws/queue.py

```diff
@@ -83,6 +83,31 @@ def get_queue(queue_name: str):
     return queue
 
 
+def list_queues(region: str):
+    """
+    Return a list of queues which the user is allowed to see
+    """
+    client = boto3.client('sqs', region_name=region)
+    queues = client.list_queues()
```

Comment:
I think this should re-use `get_queue`

```
def get_queue(queue_name: str):
    """
    Return a queue resource by name, e.g., alex-really-secret-queue
    """
    sqs = boto3.resource("sqs")
    queue = sqs.get_queue_by_name(QueueName=queue_name)
    return queue
    ```

Reply:
It does not work. 
**get_queue** uses _**resource("sqs")**_ which returns a **sqs.Queue** object not a **client**, therefore **sqs.Queue** does not have **list_queues** function, this must come from a client.

Reply:
![image](https://user-images.githubusercontent.com/25601994/130399313-e426a977-89e5-49f8-892b-ca5d879a2dc4.png)
![image](https://user-images.githubusercontent.com/25601994/130399333-5400d6ac-f441-4cbd-a60d-1655d29e4852.png)

Reply:
Ah,  you're right about this one, sorry.

Reply:
You could use a resource still, and filter, like this says: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.ServiceResource.queues

Reply:
Changed